docs/concepts/cluster-administration/admission-control-webhooks.md

```diff
@@ -0,0 +1,79 @@
+# System Administrator’s Guide To Dynamic Admission Control
+
+As a system administrator, you'll likely want to enable various admission
+plugins to enact business policies. This document covers how to do that safely.
+
+## Registering
+
+Webhooks are registered with the ValidatingAdmissionWebhook and
+MutatingAdmissionWebhook API types. At the moment, it's essential that you
+register a webhook for all versions of an API--otherwise changes to a version
+which the webhook isn't registered for won't go through the webhook. (I.e., if
+you want a webhook on Deployments, you want to register for both the
+beta version in extensions and the v1 version in apps.)
+
+## Identity and permissions
+
+### What does your webhook trust?
+
+If the webhook is going to run in your cluster, the documentation for the
+webhook should tell you how to set up the identities it will trust. If the
+webhook is an external service, the service provider will have to give you some
+mechanism for demonstrating an identity (e.g., a token or a client-cert for
+mutual TLS).
+
+The configuration file for admission controllers allows you to specify a
```

Comment:
It's worth mentioning that the configuration file is specified via the `--admission-control-config-file` flag of the apiserver. I hope that we had a doc on this config file.